dev_guide/compute_resources.adoc

```diff
@@ -331,38 +331,40 @@ implemented. There is no resource quota or limit range support for these
 resources, and they have no impact on QoS.
 ====
 
-Opaque integer resources are resources that begin with the prefix
-`pod.alpha.kubernetes.io/opaque-int-resource-`. The API server
+Opaque integer resources are called _opaque_ because {product-title} 
+does not know what the resource is, but will schedule a pod on a node 
+only if enough of that resource is available. They are called _integer resources_ 
+because they must be available, or _advertised_, in integer amounts. The API server
 restricts quantities of these resources to whole numbers. Examples of
-_valid_ quantities are `3`, `3000m` and `3Ki`. Examples of _invalid_
-quantities are `0.5` and `1500m`.
+_valid_ quantities are `3`, `3000m` and `3Ki`. 
 
-There are two steps required to use opaque integer resources. First, the cluster
-operator must advertise a per-node opaque resource on one or more nodes. Second,
-users must request the opaque resource in pods.
+Some example use cases for opaque integer resources:
 
-To advertise a new opaque integer resource, the cluster operator should
-submit a `PATCH` HTTP request to the API server to specify the available
-quantity in the `status.capacity` for a node in the cluster. After this
-operation, the node's `status.capacity` will include a new resource. The
-`status.allocatable` field is updated automatically with the new resource
-asynchronously by the kubelet.
+* LLC (last-level cache) allocation.
+* GPU devices
+* FPGA devices
+* MICs (Many-Integrated Core)
+* Slots for sharing bandwidth to a parallel file system.
+// https://github.com/kubernetes/kubernetes/issues/28312
 
-[NOTE]
-====
-Since the scheduler uses the node `status.allocatable` value when evaluating pod
-fitness, there may be a short delay between patching the node capacity with a
-new resource and the first pod that requests the resource to be scheduled on
-that node.
-====
+For example, if a node has 800 GiB of a special kind of disk storage, you could create a name for the special storage, 
+such as `_opaque-int-resource-special-storage_`. You could advertise it in chunks of a certain size, 
+such as 100 GiB. In that case, your node would advertise that it has eight resources of type opaque-int-resource-special-storage.
 
-The HTTP request below advertises five `foo` resources on node
-`openshift-node-1`. Note that `~1` is the encoding for the character `/` in the
-patch path. The operation path value in the JSON-Patch is interpreted as a
-JSON-Pointer. For more details, refer to
-link:https://tools.ietf.org/html/rfc6901#section-3[IETF RFC 6901, section 3].
+Opaque integer resource names must begin with the prefix `pod.alpha.kubernetes.io/opaque-int-resource-`. 
 
-[source,http]
+There are two steps required to use opaque integer resources. First, the cluster
+operator must advertise a per-node opaque resource on one or more nodes. Second,
+users must request the opaque resource in pods:
```

Comment:
Do the above few lines match the following procedure? I think the first part does, but the second is for users. Should the first step be for admins? We're in the dev guide, so there's probably no need to say "users must request X", because if we're talking about OpenShift users "application developers" is probably better, but also it's understood the reader is one of these, so maybe just "Second, request the...".

But if both actions are done by the developer, then I'd suggest just replacing all the above with "To use opaque integer resources:", because otherwise it's superfluous anyway.

Reply:
@bfallonf Good question on the roles. I will look into that. 
There is also a task for someone to create that resource so that it can be advertised and consumed by these two steps, which is an admin task (I am told admin types will know how to do this). Any thoughts on where I could put a reference to opaque integer resources where an admin will see it?